### ISSUE TITLE
Questions about the "checkpoint_kwargs"

### ISSUE THREAD

**Issue by yqjiang6** [saic-mdal/lama#175]
Hi!

This is a very impressive work and I try to run your code. However, a problem appeared when I try to train the model using "lama-fourier-celeba" config. I find that the problem is related to the "checkpoint_kwargs" in "any_gpu_large_ssim_ddp_final_celeba.yaml". The trainer in pytorch-lightening monitor the "val_ssim_fid100_f1_total_mean" metric, but it is not calculated in the "training_step_end" function in the model, which finally causes error when calling "self.log_dict".

Could you please help me solve this problem? The bug information is shown as following.

Thank you!
![image](https://user-images.githubusercontent.com/45167114/199229655-5c3935b7-fee8-4090-a945-dd4b97027e31.png)

**Comment by Ellohiye** [comment]
Hello! I saw your question, have you implemented this code with a pre-trained model? Have you encountered this problem?‘

(lama) root@dl-0916060380-pod-jupyter-2prjb:~/lama# python3 bin/predict.py model.path=../celebahqmodel indir=../testimg/celeba/img outdir=../output
Detectron v2 is not installed
[2022-11-02 17:15:36,769][saicinpainting.utils][WARNING] - Setting signal 10 handler <function print_traceback_handler at 0x7f4f2e195b70>
[2022-11-02 17:15:36,772][main][CRITICAL] - Prediction failed due to [Errno 2] No such file or directory: '../celebahqmodel/config1.yaml':

Configuration file not found when loading pretrained model. Looking forward to your reply！

**Comment by Ellohiye** [comment]
> 

Hello! I saw your question, have you implemented this code with a pre-trained model? Have you encountered this problem?‘

**(lama) root@dl-0916060380-pod-jupyter-2prjb:~/lama# python3 bin/predict.py model.path=../celebahqmodel indir=../testimg/celeba/img outdir=../output
Detectron v2 is not installed
[2022-11-02 17:15:36,769][saicinpainting.utils][WARNING] - Setting signal 10 handler <function print_traceback_handler at 0x7f4f2e195b70>
[2022-11-02 17:15:36,772][main][CRITICAL] - Prediction failed due to [Errno 2] No such file or directory: '../celebahqmodel/config1.yaml':**

Configuration file not found when loading pretrained model. Looking forward to your reply！

**Comment by windj007** [comment]
It seems that you turned validation off or changed the set of metrics - this metric used to be calculated after full validation  (not training) epoch in our code. To solve that, either re-enable calculation of metrics or re-configure checkointing callback to not use it.

**Comment by rose-jinyang** [comment]
Hello
How are you?
Thanks for contributing to this project.
I have the above same issue.
I checked the "best.ckpt" file is NOT outputted after several epochs in training.
It seems that the validation is NOT done.
Which parts should I check?
